src/api/treeTrackerApi.js

```diff
@@ -130,12 +131,15 @@ export default {
    * Capture Match Tool
    */
   fetchCapturesToMatch(currentPage, abortController, filter) {
-    const where = Object.keys(filter)
```

Comment:
Was this not working correctly?

Reply:
I think I ended up with a "&&" at one point, but I'll have to review it again